libs/common/src/sql-db-seeder/seeding-data-source.ts

```diff
@@ -0,0 +1,34 @@
+import { DataSource, DataSourceOptions } from 'typeorm';
+import { SeederOptions } from 'typeorm-extension';
+import path from 'path';
+import { sqlDbConfig } from '../sql-db-connection/config';
+
+const { SQL_DB_PORT, SQL_DB_USERNAME, SQL_DB_PASSWORD, SQL_DB_NAME, SQL_DB_LOGGING_ENABLED } = sqlDbConfig;
+
+const dataSourceOptions: DataSourceOptions = {
+  type: 'mssql',
+  host: 'localhost',
+  port: SQL_DB_PORT,
+  username: SQL_DB_USERNAME,
+  password: SQL_DB_PASSWORD,
+  database: SQL_DB_NAME,
+  synchronize: false,
+  logging: SQL_DB_LOGGING_ENABLED,
+  entities: [path.join(__dirname, '../sql-db-entities/**/*.entity.ts')],
+  migrations: [path.join(__dirname, '../sql-db-connection/migrations/*.ts')],
+  options: {
+    encrypt: true,
+    trustServerCertificate: true,
+  },
+};
```

Comment:
In order to redefine the `host` option for the connection, we need to specify that the `type` is `mssql`. As the original data source (exported by `../sql-db-connection`) is technically of type `DataSourceOptions`, this means that we can't simply use those options and update the `host` value as `host` only exists on certain types of the `DataSourceOptions` union